scripts/openmc-make-compton

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import tarfile
+from urllib.request import urlopen
+
+import numpy as np
+import h5py
+
+
+base_url = 'http://geant4.cern.ch/support/source/'
+filename = 'G4EMLOW.6.48.tar.gz'
+block_size = 16384
+
+# ==============================================================================
+# DOWNLOAD FILES FROM GEANT4 SITE
+
+# Establish connection to URL
+req = urlopen(base_url + filename)
```

Comment:
Same comment as above re: `urlopen` returning a HTTPResponse object (and potentially renaming `req` as response)